core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -211,6 +211,31 @@ private[spark] class AppStatusListener(
           update(rdd, now)
         }
       }
+      // remove partition information
```

Comment:
Comment does not clarify much. What partitions? Either make it clearer or remove.

Reply:
Updated about rdd's partition removed for the lost executor